cli/loader_lib.c

```diff
@@ -153,6 +152,181 @@ JL_DLLEXPORT const char * jl_get_libdir()
     return lib_dir;
 }
 
+#ifdef _OS_LINUX_
+#ifndef GLIBCXX_LEAST_VERSION_SYMBOL /* Should always be defined in the makefile. This appeases the linter. */
+#define GLIBCXX_LEAST_VERSION_SYMBOL "GLIBCXX_a.b.c"
+#endif
+
+#include <link.h>
+#include <sys/wait.h>
+
+static void write_wrapper(int fd, char *str, size_t len)
```

Comment:
IIUC, these are only used on linux, which means the unused-function warning will occur unless you hide these inside the `#if` block also